API/models/schemas/tag_schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "OGrEE Tag Schema",
+  "type": "object",
+  "properties": {
+    "slug": {
+      "type": "string",
+      "$ref": "refs/types.json#/definitions/slug"
+    },
+    "description": {
```

Comment:
Description for all others is an array of string and I don't see why, we can check with the team to make it always a string

Reply:
I think that in the rest of the cases it is to be able to put several descriptions. I don't see why we need that in a tag (neither in the others, but I guess they have done it that way for a reason...)